src/TailwindBuilder.php

```diff
@@ -48,12 +48,16 @@ public function __construct(
 
     public function runBuild(
         bool $watch,
+        bool $poll,
         bool $minify,
     ): Process {
         $binary = $this->createBinary();
         $arguments = ['-c', $this->configPath, '-i', $this->inputPath, '-o', $this->getInternalOutputCssPath()];
         if ($watch) {
             $arguments[] = '--watch';
+            if ($poll) {
+                $arguments[] = '--poll';
+            }
```

Comment:
Can't we use --poll without --watch? I see these fields are probably dependent, but this implementation will just silently not add --poll if no --watch was passed to the final command. But probably that's not a problem, because if you don't pass --watch then it's just finish the execution and that's it, so probably that's not a big deal to have this nested ot the --watch.

Just curious about your opinion on it

Reply:
In my opinion, it should not be possible to use the `--poll` option if the `--watch` option is not called, maybe we should throw an exception if this case happens ?

Reply:
Hm, that's not something that Tailwind does, so I think it's safe to merge it as is.

Thanks again!